exercises/resistor-color-duo/resistor-color-duo.test.ts

```diff
@@ -0,0 +1,23 @@
+import { value } from "./resistor-color-duo"
+
+describe('Resistor Colors', () => {
+  test('Brown and black', () => {
+    expect(value(['brown', 'black'])).toEqual(10)
+  })
+
+  test('Blue and grey', () => {
+    expect(value(['blue', 'grey'])).toEqual(68)
+  })
+
+  test('Yellow and violet', () => {
+    expect(value(['yellow', 'violet'])).toEqual(47)
+  })
+
+  test('Orange and orange', () => {
+    expect(value(['orange', 'orange'])).toEqual(33)
+  })
+
+  test('Ignore additional colors', () => {
+    expect(value(['green', 'brown', 'orange'])).toEqual(51)
```

Comment:
Can you add a couple a negative tests. What should happen when a non valid input is give. Empty input. Less than two count.

Reply:
This exercise is a level 2 exercise and probably doesn't require additional tests imo. The behaviour for less than two colours in undefined. I don't see the point in defining that behaviour.